.github/workflows/thank-you.yml

```
@@ -33,11 +33,11 @@ jobs:
 
             const { assignees, user } = result.data
 
-            const assignees_tagged = assignees.map((user) => '@' + user.login).join(' ')
-            const owner_tagged = '@' + user.login
+            const taggedUsers = assignees.length > 0 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)